wiremock/mappings/ProbationOffenderSearchApi_SearchByNomsNumber.json

```diff
@@ -0,0 +1,156 @@
+      {
+        "offenderId": 1,
+        "softDeleted": false,
+        "otherIds": {
+          "crn": "X320741",
+          "pncNumber": "2018/0123456X",
+          "croNumber": "SF80/655108T",
+          "nomsNumber": "A1234AI",
+          "previousCrn": "X10001"
+        },
+        "title": "Mr",
+        "firstName": "Aadland",
+        "middleNames": [],
+        "surname": "Bertrand",
+        "previousSurnames": [],
+        "dateOfBirth": "1978-01-06",
+        "gender": "Male",
+        "currentDisposal": "1",
+        "currentExclusion": false,
+        "currentRestriction": false,
+        "partitionArea": "some name",
+        "offenderAliases": [
+        ],
+        "contactDetails": {
+          "addresses": [
+            {
+              "from": "2018-01-16",
+              "addressNumber": "12",
+              "buildingName": "string",
+              "county": "string",
+              "district": "string",
+              "noFixedAbode": true,
+              "notes": "string",
+              "postcode": "NE2 2SW",
+              "streetName": "Church Street Lane",
+              "telephoneNumber": "string",
+              "town": "Newcastle under Lyme",
+              "status": {
+                "code":"m",
+                "description": "main"
+              },
+              "createdDateTime": "2022-08-12T10:30:27",
+              "lastUpdatedDateTime": "2022-08-12T10:30:27"
+            }
+          ],
+          "emailAddresses": [],
+          "phoneNumbers": [],
+          "allowSMS": true
+        },
+        "offenderProfile": {
+          "riskColour": "red",
+          "ethnicity": "White British",
+          "immigrationStatus": null,
+          "nationality": "British",
+          "notes": "string",
+          "offenderDetails": "string",
+          "offenderLanguages": {
+            "languageConcerns": "string",
+            "otherLanguages": [
+              "string"
+            ],
+            "primaryLanguage": "string",
+            "requiresInterpreter": true
+          },
+          "previousConviction": {
+            "detail": {}
+          },
+          "religion": "string",
+          "remandStatus": "string",
+          "secondaryNationality": "string",
+          "sexualOrientation": "string"
+        },
+        "mappa": {
+          "level": 1,
+          "levelDescription": "MAPPA Level 1",
+          "category": 2,
+          "categoryDescription": "MAPPA Category 2",
+          "startDate": "2021-02-08",
+          "reviewDate": "2021-05-08",
+          "team": {
+            "code": "NO2AAM",
+            "description": "OMIC OMU A"
+          },
+          "officer": {
+            "code": "NO2AAMU",
+            "forenames": "Unallocated",
+            "surname": "Staff"
+          },
+          "probationArea": {
+            "code": "NO2",
+            "description": "NPS London"
+          },
+          "notes": "Level 1 Cat 2 notes"
+        },
+        "probationStatus": {
```

Comment:
I think you are missing the `"currentTier"` which is at the root level (with probationStatus). A value like `"UD2"`

Reply:
also `age` at the root level, which is an integer like `30`